webapp/src/Entity/Contest.php

```diff
@@ -172,49 +172,49 @@ class Contest extends BaseApiEntity
 
     /**
      * @var boolean|null
-     * @ORM\Column(type="boolean", name="process_awards",
-     *     options={"comment"="Are there awards for this contest?","default"=1},
+     * @ORM\Column(type="boolean", name="medals_enabled",
+     *     options={"comment"="Are medals eanbled for this contest?","default"=1},
```

Comment:
Besides the typo, the comment does not add any info over the name.

Reply:
I think we have many many of these comments in the database tables.